src/styles/blocks/cards-catalog.scss

```diff
@@ -0,0 +1,113 @@
+.cards-catalog {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+
+  &:nth-child(-n+4) {
+    margin-top: 49px;
```

Comment:
This margin should be 50px instead of 49 because the borders are not included in the design